lib/chef/data_collector/run_end_message.rb

```diff
@@ -133,7 +133,15 @@ def action_record_for_json(action_record)
           end
 
           hash["conditional"] = action_record.conditional.to_text if action_record.status == :skipped
-          hash["error_message"] = action_record.exception.message unless action_record.exception.nil?
```

Comment:
Deliberately remove `error_message` attribute as going to add more error info.

Reply:
That probably breaks A2 though.  Even if it doesn't this is a public API that anyone can use, and that breaks backwards compatibility of the API.  This needs to be backed out and included.